<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TetrisAttackClone</class>
 <widget class="QMainWindow" name="TetrisAttackClone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tetris Attack Clone</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>801</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
    </property>
    <property name="text">
     <string>Tetris Attack Clone</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="highScoresButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
    </property>
    <property name="text">
     <string>High Scores</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="howToPlayButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
    </property>
    <property name="text">
     <string>How To Play</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="gameresources.qrc">:/screenshots/background.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>Title</zorder>
   <zorder>startButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>highScoresButton</zorder>
   <zorder>howToPlayButton</zorder>
   <zorder>aboutButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gameresources.qrc"/>
 </resources>
 <connections/>
</ui>
